### ISSUE TITLE
Communication Entry Throws Error When Email and Lava are Disabled

### ISSUE THREAD

**Issue by courtneycooksey** [SparkDevNetwork/Rock#4239]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

**Note: I could not duplicate this on the demo site because communications are not configured on the demo site**

### Description

We have a page in Rock that only contains the Communication Entry block and is used strictly for sending SMS messages. When I disable email as an available medium on the block you can no longer send the SMS messages if lava is also disabled. 

It happens in this section 
```
            // If we are not allowing lava then remove the syntax
            if ( !GetAttributeValue( AttributeKey.EnableLava ).AsBooleanOrNull() ?? false )
            {
                communication.Message = communication.Message.SanitizeLava();
                communication.Subject = communication.Subject.SanitizeLava();
                communication.BCCEmails = communication.BCCEmails.SanitizeLava();
                communication.CCEmails = communication.CCEmails.SanitizeLava();
                communication.FromEmail = communication.FromEmail.SanitizeLava();
                communication.FromName = communication.FromName.SanitizeLava();
                communication.ReplyToEmail = communication.ReplyToEmail.SanitizeLava();
            }
```
Since the message isn't an email message and email isn't an available option the values are all null, when the SanitizeLava() function is called in StringHtmlExtensions.cs it breaks because it can't handle null values. 

When email is an available medium for the block all the above values are initialized to empty strings so the error does not occur and when you enable lava it skips the section entirely so the error does not occur then either. 

### Steps to Reproduce

1. Create a page containing the Communication Entry block
2. Only enable SMS as a Medium and do not enable Lava
3. Try to send a communication

**Expected behavior:**

SMS Communication Sends. You see a confirmation screen. 

**Actual behavior:**

Error is thrown, communication does not send. 

### Versions

* **Rock Version:** Rock McKinley 10.2 (1.10.2.3)
* **Client Culture Setting:** en-US

**Comment by cabal95** [comment]
Thanks for reporting @courtneycooksey, this appears to be a duplicate of #4175, which is fixed in 10.3. I verified the error on the demo site, and also verified that it works fine on the prealpha site so this does indeed seem to be fixed in future versions.

**Comment by courtneycooksey** [comment]
My bad, I didn't realize I was only looking through open issues when I was looking for similar ones.
